guides/source/caching_with_rails.md

```diff
@@ -304,6 +304,8 @@ end
 
 NOTE: Notice that in this example we used the `cache_key` method, so the resulting cache key will be something like `products/233-20140225082222765838000/competing_price`. `cache_key` generates a string based on the model's `id` and `updated_at` attributes. This is a common convention and has the benefit of invalidating the cache whenever the product is updated. In general, when you use low-level caching for instance level information, you need to generate a cache key.
 
+NOTE: Since Rails 5.2, the `cache_key` method returns a stable key without timestamp in default. To get more details see the API Documentation for [ActiveRecord::Integration](https://api.rubyonrails.org/classes/ActiveRecord/Integration.html#method-i-cache_key).
```

Comment:
Since the documentation is versioned for 6.0, I think we can probably just remove the timestamp in the previous note instead of specifying this. We might want to backport this change to `5-2-stable` once this is merged.

Reply:
@gmcgibbon
Thank you for reviewing. I removed the timestamp in the previous note and change pull request title, summary.